UnitsNet.Tests/CustomCode/VolumeFlowTests.cs

```diff
@@ -109,6 +109,10 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double MegaukGallonsPerSecondInOneCubicMeterPerSecond => 2.19969e-4;
 
+        protected override double MegaukGallonsPerDayInOneCubicMeterPerSecond => 19.005304;
```

Comment:
19.00534305304118 M UK gal/day (mega UK gallons per day)
https://www.wolframalpha.com/input?i=1.0000000000000000+Cubic+Meter+Per+Second+to+Megaukgallons+Per+Day

Reply:
Fixed.